common/lock.hpp:63: error: 'NULL' was not declared in this scope

--- common/lock.hpp.orig	Tue Jan 17 11:20:03 2012
+++ common/lock.hpp	Tue Jan 17 11:20:13 2012
@@ -16,6 +16,7 @@
 #define DISTRIBNET_LOCK__HPP
 
 #include <assert.h>
+#include <cstddef>
 
 #include "settings.h"
 
